### PR TITLE
List Greenhouse to Defi Llama

### PR DESCRIPTION
**Pull request by GreenFlorist**
##### Twitter Link:
https://twitter.com/GREENHOUSEDEX

##### Website Link:
https://greenhousedex.com/

##### Logo (High resolution, preferably in .svg and .png, for application on both white and black backgrounds. Will be shown with rounded borders):
https://greenhousedex.com/images/green.png

##### Current TVL:
$526,612
##### Chain:
Polygon

##### Coingecko ID (so your TVL can appear on Coingecko): (https://api.coingecko.com/api/v3/coins/list)
greenhouse

##### Coinmarketcap ID (so your TVL can appear on Coinmarketcap): (https://api.coinmarketcap.com/data-api/v3/map/all?listing_status=active,inactive,untracked&start=1&limit=10000)
greenhouse

##### Short Description (to be shown on DefiLlama):
A sustainable multi-chain DEX which builds and drives liquidity for layer-one blockchains, bridges and partnering projects.

##### Token address and ticker if any:
0x40DB6d7812b8288eCA452F912ca9F262b186f278 
Symbol: GREEN
Name: Greenhouse

##### Category (full list at https://defillama.com/categories) *Please choose only one:
[Dexes](https://defillama.com/protocols/dexes)

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/greenhouse/index.js exports TVL: 
        
 
 
    polygon                   542.92 k
    
    total                    542.92 k

File: projects/greenhouse/index.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
+const sdk = require("@defillama/sdk");
+const erc20 = require("../helper/abis/erc20.json");
+const abi = require("./abi.json");
+const { unwrapUniswapLPs } = require("../helper/unwrapLPs");
+const { transformPolygonAddress } = require("../helper/portedTokens");
+
+const MASTERCHEF_CONTRACT = "0xbD40a260Ddd78287ddA4C4ede5880505a9fEdF9a";
```
Comment (g1nt0ki) [inline projects/greenhouse/index.js:7]: @waynebruce0x  is `addFundsInMasterChef` applicable for this adapter?

Comment (waynebruce0x) [reply projects/greenhouse/index.js:7]: yesss I think that's a good suggestion, thanks g1nt0ki

**Comment by g1nt0ki** [comment]
@GreenFlorist I put up another PR that has this adapter but with masterchef helper.

https://github.com/DefiLlama/DefiLlama-Adapters/pull/2123.

I am closing this one in favor of that

**Pull request closed without merging**